environment_queries/duration.tf

```diff
@@ -19,9 +9,6 @@ data "honeycombio_query_specification" "heatmap_duration" {
 
   time_range = var.query_time_range
 
-  depends_on = [
```

Comment:
hey @tdarwin  I'm not super familiar with this codebase - but looking through most of the removed depends_on statements appear to be for dataset `__all__` so I'm guessing order doesn't matter.   But this one caught my eye, because it seems like it could fail if the column doesn't exist -- right?

Reply:
So, [the dependency](https://github.com/honeycombio/terraform-honeycombio-opentelemetry-starter-pack/blob/e680eb1c2ec85ed0adfe005753b1c8ad2199972b/main.tf#L55) is actually setup in the main.tf of the parent module now because [the column creation](https://github.com/honeycombio/terraform-honeycombio-opentelemetry-starter-pack/blob/e680eb1c2ec85ed0adfe005753b1c8ad2199972b/main.tf#L13-L41) needed to be more organized.

So I pulled out all the dependencies, because the columns aren't being created with the queries. Instead, I ensure all columns are created before the module even starts to try to create derived columns or queries at all.

Reply:
ah okay, cool!